Resources/Prototypes/Roles/Jobs/Command/captain.yml

```diff
@@ -6,16 +6,19 @@
   requirements:
     - !type:DepartmentTimeRequirement
       department: Engineering
-      time: 54000 # 15 hours
+      time: 18000 # 5 hours
     - !type:DepartmentTimeRequirement
       department: Medical
-      time: 54000 # 15 hours
+      time: 18000 # 5 hours
     - !type:DepartmentTimeRequirement
-      department: Security
-      time: 54000 # 15 hours
+      department: Science
+      time: 18000 # 5 hours
+    - !type:DepartmentTimeRequirement
+      role: Security
+      time: 18000 # 5 hours
     - !type:DepartmentTimeRequirement
       department: Command
-      time: 54000 # 15 hours
+      time: 36000 # 10 hours
```

Comment:
I say reduce all of these to 4 so it's max 20 hours.
Command roles are limited to 1 person so someone could be unfortunate and take a long time to get command. So having it roughly 2-4 rounds (depending how you see it) makes it more fair.